sys/net/if_bridge.c

```diff
@@ -916,12 +916,14 @@ bridge_ioctl(struct ifnet *ifp, u_long cmd, caddr_t data)
 		bc = &bridge_control_table[ifd->ifd_cmd];
 
 		if (cmd == SIOCGDRVSPEC &&
-		    (bc->bc_flags & BC_F_COPYOUT) == 0) {
+		    (bc->bc_flags & BC_F_COPYOUT) == 0 &&
```

Comment:
You could write these as:

```
if (cmd == SIOCGDRVSPEC &&
   (bc->bc_flags & (BC_F_COPYOUT | BC_F_COPYOUTCAP)) == 0) {
```

That might be easier to read?  It would have avoided the `&&` vs `||` typo earlier?